volume/store/store.go

```diff
@@ -194,12 +194,14 @@ func (s *VolumeStore) create(name, driverName string, opts map[string]string) (v
 		}
 	}
 
-	logrus.Debugf("Registering new volume reference: driver %q, name %q", driverName, name)
```

Comment:
I like that the log-message is better, but perhaps we still want to have a log message *before* volumedrivers.GetDriver() to see where something went wrong? idk